pkg/kubehunter/scanner.go

```diff
@@ -25,22 +25,22 @@ import (
 )
 
 const (
-	kubeHunterVersion       = "0.3.1"
 	kubeHunterContainerName = "kube-hunter"
 )
 
-var (
-	kubeHunterContainerImage = fmt.Sprintf("aquasec/kube-hunter:%s", kubeHunterVersion)
-)
+// KubeHunterVersion version of kube-hunter being used
+var KubeHunterVersion string
```

Comment:
We should take version from the imageRef and avoid using global state / package level variables.